fiaas_mast/metadata_generator.py

```diff
@@ -45,8 +45,11 @@ def merge_tags(self, generator_object, config):
     def metadata(self, generator_object, namespace, deployment_id):
         application_name = generator_object.application_name
         labels = {"fiaas/deployment_id": deployment_id, "app": application_name}
+        annotations = {}
+        if generator_object.spinnaker_application != "":
+            annotations["moniker.spinnaker.io/application"] = generator_object.spinnaker_application
         # TODO: Why doesn't annotations default to a dict?
-        metadata = ObjectMeta(name=application_name, namespace=namespace, labels=labels, annotations={})
+        metadata = ObjectMeta(name=application_name, namespace=namespace, labels=labels, annotations=annotations)
 
         if generator_object.spinnaker_tags:
             metadata.annotations = self.spinnaker_annotations(generator_object)
```

Comment:
if we have both `spinnaker_application` and `spinnaker_tags`, the annotation from `spinnaker_application` is overwritten here. Is that what we want?

Reply:
thank you will take a look